pkg/agent/datamodel/types.go

```diff
@@ -1226,6 +1226,33 @@ func (config *NodeBootstrappingConfiguration) GetOrderedKubeletConfigStringForPo
 	return strings.TrimSuffix(buf.String(), ", ")
 }
 
+// GetOrderedKubeproxyConfigStringForPowershell returns an ordered string of key/val pairs for Powershell script consumption
+func (config *NodeBootstrappingConfiguration) GetOrderedKubeproxyConfigStringForPowershell() string {
+	// https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/
+	// --metrics-bind-address ipport     Default: 127.0.0.1:10249
+	// 	The IP address with port for the metrics server to serve on (set to '0.0.0.0:10249' for all IPv4 interfaces and '[::]:10249' for all IPv6 interfaces). Set empty to disable.
+	// This only works with Windows provisioning package v0.0.15+.
+	// https://github.com/Azure/aks-engine/blob/master/docs/topics/windows-provisioning-scripts-release-notes.md#v0015
+	if config.KubeproxyConfig == nil {
```

Comment:
did we check all k8s versions support this arg?

Reply:
From docs, it has been enabled for a long long time. At least k8s 1.19+ support it.

Reply:
https://github.com/kubernetes/kubernetes/blob/v1.19.0/cmd/kube-proxy/app/server.go#L170

Reply:
means you will only set this option for the 1.19+?

Reply:
I set it for all versions but AKS only supports 1.19+ today

Reply:
will add agent pool or scale agent pool hit this code path?

Reply:
or rotate cert.

Reply:
vmss model is only updated in create/upgrade operations. Scale does not update vmss model so it will not hit this path

Reply:
how about rotate cert?

Reply:
I have checked that v1.13.0+ support this parameter.